### PR TITLE
SRE-76 move PHP classes from config to app

### PR DESCRIPTION
**Pull request by ghost**
https://wikia-inc.atlassian.net/browse/SRE-76

move PHP classes from config to app

* class FullRequestLogger from config/Logging.php
* class AuditLog from config/AuditLog.php
* $wgHooks assignments from CommonExtensions

https://github.com/Wikia/config/pull/2355

@drozdo @ljagiello @TK-999 @macbre

### CODE REVIEW

File: includes/wikia/FullRequestLogger.class.php [hunk 1]
```diff
@@ -0,0 +1,166 @@
+<?php
+
+class FullRequestLogger {
```
Comment (mszabo-wikia) [inline includes/wikia/FullRequestLogger.class.php]: Let's verify if this is still needed. In the era of global auth pages, `UserLoginSpecialController` is no longer used, and the Digital Ocean IPs may well be out of date. We might also have other sources for such access logs.

File: includes/wikia/AuditLog.class.php [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+					'new_email_old' => $oldUser->getNewEmail(),
+					'new_email_new' => $user->getNewEmail(),
+				] );
+		}
+
+		return true;
+	}
+
+	static public function onAddNewAccount() {
+		return true;
```
Comment (mszabo-wikia) [inline includes/wikia/AuditLog.class.php]: redundant hook handler

**Review by mszabo-wikia — CHANGES_REQUESTED** [review]

File: includes/wikia/FullRequestLogger.class.php (hunk 1 above)
Comment (ljagiello) [reply includes/wikia/FullRequestLogger.class.php]: Why do we even have hardcoded IPs there? It sounds wrong.

Comment (ljagiello) [reply includes/wikia/FullRequestLogger.class.php]: It looks like it was introduced back in security incident days I think we should do at CDN level not inside application.

Comment (ghost) [reply includes/wikia/FullRequestLogger.class.php]: Okay, I will review it and remove if no longer needed.

**Comment by ghost** [comment]
I have removed `FullRequestLogger` completely, we are going to have ca. 3 million entries per day less in our logs. :)

**Review by ljagiello — APPROVED** [review]

**Comment by ghost** [comment]
Removing obsolete hooks from `CheckUser` is a bonus.

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**